gridstatus/isone_api/isone_api.py

```diff
@@ -644,8 +644,111 @@ def _handle_load_forecast(
         )
         return df[
             list(
-                regional_cols.values()
-                if "Location" in df.columns
-                else system_cols.values(),
+                (
+                    regional_cols.values()
+                    if "Location" in df.columns
+                    else system_cols.values()
+                ),
             )
         ]
+
+    @support_date_range("DAY_START")
+    def get_interchange_hourly(
+        self,
+        date: str | pd.Timestamp = "latest",
+        end: str | pd.Timestamp | None = None,
+        verbose: bool = False,
+    ) -> pd.DataFrame:
+        """
+        Get the hourly interchange data for specified date range. Hourly data includes
+        multiple locations.
```

Comment:
The unique locations are: '.I.HQHIGATE120 2', '.I.HQ_P1_P2345 5', '.I.NRTHPORT138 5', '.I.ROSETON 345 1', '.I.SALBRYNB345 1', '.I.SHOREHAM138 99' which match the [`interfaces` in ISONE.py](https://github.com/gridstatus/gridstatus/blob/f8116105b42cc71197aa488e3bb382507b1199e6/gridstatus/isone.py#L50-L57).